installing/installing-mirroring-installation-images.adoc

```diff
@@ -84,7 +84,35 @@ include::modules/olm-mirroring-catalog.adoc[leveloffset=+1]
 
 * xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
 
-include::modules/olm-mirroring-catalog-prereqs.adoc[leveloffset=+2]
+[id="olm-mirror-catalog-prerequisites_installing-mirroring-installation-images"]
+=== Prerequisites
+
+Mirroring Operator catalogs for use with disconnected clusters has the following prerequisites:
```

Comment:
```suggestion
Using mirroring Operator catalogs with disconnected clusters has the following prerequisites:
```

Just a small suggestion that may make it a little easier to read. Please ignore, if it changes the meaning of the sentence though.

Reply:
Yeah, changes it a bit.